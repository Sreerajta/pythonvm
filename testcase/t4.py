a=25
b=10

if(a<b):
 print "a less than b"
else:
 print "else condition : b less than a"
