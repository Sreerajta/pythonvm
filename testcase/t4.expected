else condition : b less than a
